graphics/src/AssimpLoader.cc

```diff
@@ -49,6 +49,9 @@ class AssimpLoader::Implementation
   /// Convert a color from assimp implementation to Ignition common
   public: math::Color ConvertColor(aiColor4D& _color);
 
+  // Stores the loaded bone names 
+  public: std::vector<std::string> BoneNames;
```

Comment:
Since the only operations we do on this structure are insertion and lookup by name I'd recommend using a `std::unordered_set<std::string>` so lookup can be in (approximately) constant time. Using a vector forces us to use `std::find` that will have to traverse the whole structure to find a key.

Also variable names usually have the `lowerCamelCase` convention

Reply:
Done, thanks for heads up.